.github/workflows/ota-testbench.yml

```
@@ -0,0 +1,29 @@
+name: OTA test bench
+on:
+  push:
+  pull_request:
+  workflow_call:
+
+jobs:
+  ota-testbench:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)